prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -73,75 +133,179 @@ <h2>BC Services Card</h2>
             <br>
 
             <a [href]="bcscMobileSetupUrl"
-               target="_blank"
-               rel="noopener noreferrer">
-              How to set up the BC Services Card app.
-            </a>
+                 target="_blank"
+                 rel="noopener noreferrer">
+                How to set up the BC Services Card app.
+              </a>
           </p>
         </section>
-
-        <div class="info-graphic">
-          <img src="/assets/bc_services_card.svg"
-               alt="BC Services Card Icon">
-        </div>
       </div>
-
     </div>
-    <div class="col-sm-12 col-md-4 steps step-2">
 
-      <div class="step-container">
-        <section class="content has-step-background">
+    <div class="col-sm-12 col-md-4"
+         *ngIf="mode === 'enrolment'">
+      <div>
+        <div class="graphic-wrapper">
+          <img class="graphic"
+               src="/assets/enrolment_v2.svg"
+               alt="Enrolment Icon">
+        </div>
+        <section class="content">
           <h2>PRIME Enrolment</h2>
 
           <p>
-            Complete PRIME enrolment by answering a set of questions. It usually only takes a few minutes. If you are a
-            registrant of a B.C. health college, you will need your college information (licence number/ registration
+            Complete PRIME enrolment. It usually only takes a few minutes. If you are a
```

Comment:
I believe Business wanted "Complete **your** PRIME enrolment"

Reply:
It is fixed.